### PR TITLE
Ulex camlp5

### PR DESCRIPTION
**Pull request by sacerdot**
- ulex-camlp5 has been ported to ocaml 5.0.0 & friends
- version bumped to 1.4

### CODE REVIEW

File: opam [hunk 1]
```diff
@@ -1,8 +1,9 @@
 opam-version: "2.0"
 name: "ulex-camlp5"
-version: "1.2"
+version: "1.4"
```
Comment (whitequark) [inline opam]: Why skip 1.3?

Comment (sacerdot) [reply opam]: You are right, no reason for that. I probably had already bumped it in my copy in the past and I forgot.

I have added a new commit to use 1.3.

Thanks!

Comment (sacerdot) [reply opam]: In the meantime I have released ulex-camlp5-1.3 in opam. Even the last version of the package (1.2) used to point to my fork, even if I would like to keep it in sync with your camlp5 branch.

Comment (sacerdot) [reply opam]: Note: I tested and the opam file can be changed to accept ocaml >= 4.14.1 too!

**Pull request merged**